Podcast/EpisodeTableViewCell.swift

```diff
@@ -53,10 +53,6 @@ class EpisodeTableViewCell: UITableViewCell, EpisodeSubjectViewDelegate {
         episodeSubjectView.episodeUtilityButtonBarView.setBookmarkButtonToState(isBookmarked: isBookmarked)
     }
     
-    func setPlayButtonToState(isPlaying: Bool) {
```

Comment:
Why is this removed?

Reply:
addressed .. I removed it because we now need to update the slider value as well so we need more than just the isPlaying boolean.